src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, FigureDraw {
+    private Color color;
+    private String name;

```

Comment:
remove this field, you don't need it, if you want to add name value to draw method, add it directlye